src/fieldValidators.js

```diff
@@ -86,7 +86,7 @@ const testEmailFormat = (value: ?string) => {
 const testLinkedInUrlFormat = (value: ?string) => {
     if (!value) return null;
 
-    const regexp = /(https?)?:?(\/\/)?(([w]{3}||\w\w)\.)?linkedin.com\/in\//; // eslint-disable-line no-control-regex
+    const regexp = /(https?):(\/\/)(([w]{3}||\w\w)\.)?linkedin.com\/in\/([a-zA-Z0-9-#?$%&=_+])+/; // eslint-disable-line no-control-regex
```

Comment:
Pourquoi avoir ajouté une partie à la fin ?

Reply:
Parce que le validator accepterait ce format `https://www.linkedin.com/in/` ce qui ne renverrai pas vers un profile. Du coup je me suis dit qu'on pourrait forcé à mettre obligatoirement quelque chose après pour éviter les erreurs.

Reply:
Pour éviter de se rajouter trop de contrainte et garder ton idée, je pense que 
```suggestion
    const regexp = /(https?):(\/\/)(([w]{3}||\w\w)\.)?linkedin.com\/in\/.+/; // eslint-disable-line no-control-regex
```
suffirait